src/Compilers/CSharp/Test/Emit/CodeGen/IndexAndRangeTests.cs

```diff
@@ -280,70 +280,70 @@ public static void Main()
             var verifier = CompileAndVerify(comp, expectedOutput: "bc");
             verifier.VerifyIL("C.M", @"
 {
```

Comment:
nit: I would have expected the IL to be the same (modulo method names). Seems like the ordering changed. It's probably not worth investigating, but seems a bit curious.

Reply:
Ah, this is due to the move from a static Create method to a constructor call. Struct constructors can feature either a `call` or a `newobj` instruction, which differ on whether they initialize already allocated memory, or allocate memory themselves. Our code generator tries to decide which is the most efficient, so there may be a few non-local changes to the IL as the construction is rewritten.

Reply:
Makes sense. Thanks